vyper/codegen/expr.py

```diff
@@ -265,6 +265,14 @@ def parse_Attribute(self):
                 return IRnode.from_list(["callvalue"], typ=UINT256_T)
             elif key == "msg.gas":
                 return IRnode.from_list(["gas"], typ=UINT256_T)
+            elif key == "block.blobbasefee":
+                if not version_check(begin="cancun"):
```

Comment:
this should throw an exception, not warn

Reply:
What is the proper way to do that? It seems like everything else just warns on wrong version?